packages/api/src/entity/library_item.ts

```diff
@@ -58,7 +58,7 @@ export class LibraryItem {
   @Column('text')
   originalUrl!: string
 
-  @Column('text', { nullable: true })
+  @Column('text', { nullable: true, select: false })
```

Comment:
Can you tell me what this does? It looks like iOS expects this, but I think we handle it in the resolver now

Reply:
I don't think we use it at all. We planned to use it for downloading